app/controllers/projects_controller.rb

```diff
@@ -4,7 +4,7 @@ class ProjectsController < BaseController
   before_action :find_project, except: [:edit_collection]
 
   def index
-    @projects = current_user.projects
+    @projects = current_user.project_users.is_not_archived.map { |project| Project.find(project.project_id) }
```

Comment:
1. I like naming `unarchived` to replace `is_not_archived `
2. I think u need to call one more scope to sort it. Such as `current_user.project_users.unarchived.sorted`
3. `current_user.project_users.map(&:project)` can represent you `map` code block.